test/exe/require_static.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copied from https://github.com/envoyproxy/envoy/blob/a12869fa9e9add4301a700978d5489e6a0cc0526/test/exe/envoy_static_test.sh.
+
+if [[ $(uname) == "Darwin" ]]; then
+  echo "macOS doesn't support statically linked binaries, skipping."
+  exit 0
+fi
+
+# We can't rely on the exit code alone, since ldd fails for statically linked binaries.
```

Comment:
Yeah I noticed that as well when using `ldd` to check a go program. it instead complains the input file is not valid :)